packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-tooltip.tsx

```diff
@@ -0,0 +1,99 @@
+import { Component, Element, h, Prop } from '@stencil/core';
```

Comment:
is there someway we can leverage the existing tooltip? is there a reason we need to create a new one?

Reply:
I guess the existing tooltip 'sc-grid-tooltip' is for grid. And according to UX, there are some differences in the input and presentation of the existing tooltip and the new tooltip, like color display and removing 'Status' text ....